### DISCUSSION TITLE
LinkedIn Learning Downloader v1.1

### DISCUSSION THREAD

**Discussion by M0r0cc4nGh0st** [M0r0cc4nGh0st/LinkedIn-Learning-Downloader#1] [Announcements]
## Features
- **Sequential and Parallel Downloading:** Choose between sequential and simultaneous downloading of up to five courses or one Path link, use the parallel mode with _precaution_.
- **Exercise Files:** Upgraded scripts allow for retrieval of exercise files, also fix the bug if the course doesn't have them, offering a complete learning package for users.
- **Numeration System:** Downloads are organized with a clear numeration system for easy reference, improving usability.
- **Robust Error Handling:** Advanced error handling ensures smooth operation across a wide range of course URLs, reducing interruptions and user issues.
- **Universal URL Support:** Accepts all types of LinkedIn Learning course URLs, providing unparalleled flexibility in course downloads.
- **Abort Downloading:** Added a checkbox to abort downloading if it takes too long or encounters problems.
- **UI Design:** Revamped the UI to look more modern.

![Screenshot 2024-05-17 010941](https://github.com/M0r0cc4nGh0st/LinkedIn-Learning-Downloader/assets/157702486/fd4f9efd-7679-41c2-8414-e58c590ce2a3)

<hr /><em>This discussion was created from the release <a href='https://github.com/M0r0cc4nGh0st/LinkedIn-Learning-Downloader/releases/tag/TagsV1.1'>LinkedIn Learning Downloader v1.1</a>.</em>

**Comment by fedefabiancastro** [comment]
Saludos. Hago todo lo que me indican en las instrucciones pero no logro descargar nada. Ayuda. Gracias

**Reply by M0r0cc4nGh0st** [reply]
¿Usaste el código de scripting o la herramienta ejecutable?

Perdón por tardar en responderte, estaba de viaje, de todas formas espero tu respuesta
